pour/web/src/status-reading.svelte

```diff
@@ -26,6 +41,22 @@
       clearInterval(interval);
     }
   });
+
+  $: displayStatus = (() => {
+    if (!status) {
+      return "unknown";
+    }
+
+    for (const key in displayStates) {
+      if (displayStates.hasOwnProperty(key)) {
+        const values = displayStates[key];
+        if (values.some((val) => status.includes(val))) {
```

Comment:
Why are we looking up by value instead of key? Feels like the map is backwards

Reply:
Because the map is backwards is the right answer : ) 
I'll look into it

Reply:
Oh yea I remember now. The code is structured this way because the status from the robot might contain more information than just a single, specific state. It seemed better this way but lmk if I need to change.

Reply:
It's fine :)